src/ctelnet.cpp

```diff
@@ -2533,9 +2531,7 @@ void cTelnet::slot_processReplayChunk()
                 }
             }
 
-            if (mUSE_IRE_DRIVER_BUGFIX || mLF_ON_GA) {
```

Comment:
This changes behaviour, doesn't it?

Reply:
`mLF_ON_GA` is always `true` - well *after* a connection has been made *once* so either the behaviour in a replay is ill-defined (depends on whether a connection has previously been made in the session) or irrelevant.

Ah,  further examination shows that `recvdGA` is reset **before** the loop that contains the `if (recvdGA) {` on old|new line 2527|2525 - ***and it is not changed by anything inside the loop*** - so the whole of this `if` is *unreachable* anyhow...

Reply:
Ah, no, that was a premature assertion - `processTelnetCommand(command)` is called and that CAN change the state of `recvdGA`...

Reply:
How about the mUSE_IRE_DRIVER_BUGFIX option too?

Please test with IRE games and a few others to make sure this doesn't break anything 🙂

Reply:
As `mLF_ON_GA` is set to true as part of the connection process (and never reset afterwards) the state of `mUSE_IRE_DRIVER_BUGFIX` is irrelevant: the code in the `if (mUSE_IRE_DRIVER_BUGFIX || mLF_ON_GA) {` ... `}` will always be executed! **Remember this section of code is involved in the playback of the data of a recording NOT that from a live game.** The behaviour in the corresponding bit of the live game data processing does not consider the BUGFIX flag - mind you that flag is considered in the `(void) cTelnet::gotPrompt(std::string&)` method which IS used in processing both data streams.

Incidentally, that call to `gotPrompt(...)` is not, unreasonably, blocked if the `mFORCE_GA_OFF` is set - for the live data that is, and only for that. The replay data case always does the extra processing of that method if a GA is detected - even if it is supposed to be disabled.